lang/en.json

```diff
@@ -163,6 +163,9 @@
         "DiffDeleted": "Deleted",
         "DisablePreview": "Disable Preview",
         "DisablingPreviewPleaseWait": "Disabling preview mode, please wait...",
+        "UpdatingDebugSiteUrlPleaseWait": "Updating debug site URL, please wait...",
+        "DebugUrlNoUrlErrorMessage": "Please Enter a URL to start debugging",
+        "DebugUrlSameUrlErrorMessage": "Already debugging the same site, please visit the %1$ssite%2$s or enter new url to debug",
```

Comment:
fyi here is a dot missing at the end. Same above in "DebugUrlNoUrlErrorMessage" and possibly others.

Reply:
Updated